apps/baseline/viewsets.py

```diff
@@ -1550,3 +1554,207 @@ class CopingStrategyViewSet(BaseModelViewSet):
+
+      - slice_strategy_type - you can specify multiple, and you need to pass the code not the label (which could be
+        translated). (These are case-insensitive but otherwise must be an exact match.)
+
+    The slice is defined by matching any of the products, AND any of the strategy types (as opposed to OR).
+
+    Translated fields, eg, name, description, are rendered in the currently selected locale if possible. (Except
+    Country, which has different translations following ISO.) This can be selected in the UI or set using eg,
+    &language=pt which overrides the UI selection.
+
+    You select the fields you want using the &fields= parameter in the usual way. If you omit the fields parameter all
+    fields are returned. These are currently the same field list as the normal LZB endpoint, plus the aggregations,
+    called slice_sum_kcals_consumed, sum_kcals_consumed, kcals_consumed_percent, plus product CPC and product common
+    name translated. If you omit a field, the statistics for that field will be aggregated together.
+
+    The ordering code is also shared with the normal LZB endpoint, which uses the standard &ordering= parameter. If
+    none are specified, the results are sorted by the aggregations descending, ie, biggest percentage first.
+
+    Example URL:
+
+    http://localhost:8000/api/livelihoodzonebaselinereport/
+        ?language=pt
+        &slice_product=R09
+        &slice_strategy_type=MilkProduction
+        &fields=id,name,description,source_organization,source_organization_name,livelihood_zone,livelihood_zone_name,
+            country_pk,country_iso_en_name,main_livelihood_category,bss,currency,reference_year_start_date,
+            reference_year_end_date,valid_from_date,valid_to_date,population_source,population_estimate,
+            livelihoodzone_pk,livelihood_strategy_pk,strategy_type,livelihood_activity_pk,wealth_group_category_code,
+            product_cpc,product_common_name
+        &source_organization=1
+        &min_kcals_consumed_percent=52
+        &max_kcals_consumed_percent=99
+
+    The strategy type codes are:
+        MilkProduction
+        ButterProduction
+        MeatProduction
+        LivestockSale
+        CropProduction
+        FoodPurchase
+        PaymentInKind
+        ReliefGiftOther
+        Hunting
+        Fishing
+        WildFoodGathering
+        OtherCashIncome
+        OtherPurchase
+
+    The product hierarchy can be retrieved from the classified product endpoint /api/classifiedproduct/.
+
+    You can then filter by the percentage of your slice. The only value we have data for so far is kcals_consumed,
+    filtered using, eg, &min_kcals_consumed_percent=52&max_kcals_consumed_percent=99.
+
+    The API currently only supports a single slice at a time. For combining please run multiple searches, and add
+    the desired results to the Compare tab.
+    """
```

Comment:
I think it would be useful if the docstring included an example response, e.g. a JSON one.